scripts/airdropBsxToHdx/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "scripts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "getHdxAirdropData": "ts-node getData.ts",
+    "getHdxAirdropVestSchedules": "ts-node getVestingSchedules.ts",
```

Comment:
@timothydm42 how do you get this running?

```
airdropBsxToHdx bsx-airdrop+ $ yarn getHdxAirdropVestSchedules   
yarn run v1.22.17
$ ts-node getVestingSchedules.ts
Error: Cannot find module 'BN.js'
```

there is not any dependency in package json. am I doing something wrong?

Reply:
Sorry about that @lumir-mrkva ,  

I've deleted my local deps,  fixed `package.json` with the deps it should have had , installed, then run the scripts and they're working for me now so you should be able to run them as well

Reply:
thats weird, it is still failing for me. i have tried to add bn.js myself as a dependency but it didn't resolve my issue. still failing on `MODULE_NOT_FOUND` for some reason

Reply:
works on my mac fine, but nothing I am doing is helping my linux machine